src/python_testing/TC_MCORE_FS_1_3.py

```diff
@@ -15,15 +15,31 @@
 #    limitations under the License.
 #
 
-# This test requires a TH_SERVER application that returns UnsupportedAttribute when reading UniqueID from BasicInformation Cluster. Please specify with --string-arg th_server_app_path:<path_to_app>
+# This test requires a TH_SERVER_NO_UID application that returns UnsupportedAttribute
+# when reading UniqueID from BasicInformation Cluster. Please specify the app
+# location with --string-arg th_server_no_uid_app_path:<path_to_app>
 
+# See https://github.com/project-chip/connectedhomeip/blob/master/docs/testing/python.md#defining-the-ci-test-arguments
+# for details about the block below.
+#
+# === BEGIN CI TEST ARGUMENTS ===
+# test-runner-runs: run1
+# test-runner-run/run1/app: examples/fabric-admin/scripts/fabric-sync-app.py
+# test-runner-run/run1/app-args: --app-admin=${FABRIC_ADMIN_APP} --app-bridge=${FABRIC_BRIDGE_APP} --stdin-pipe=dut-fsa-stdin --discriminator=1234
+# test-runner-run/run1/factoryreset: True
+# test-runner-run/run1/script-args: --storage-path admin_storage.json --commissioning-method on-network --discriminator 1234 --passcode 20202021 --string-arg th_server_no_uid_app_path:${LIGHTING_APP_NO_UNIQUE_ID}
+# test-runner-run/run1/script-start-delay: 5
```

Comment:
No other scripts seem to use this script-start-delay option. Could we avoid it somehow?

Any fixed-time delay is not ideal, either it is too long because apps start fast or too short and makes tests flaky. Being able to have conditional tests is better.

Reply:
In this case we can't script-start-delay was added for this test. Every other DUT application is just a single app, but fabric sync is two applications that needs to do some level of initial configuration. The CI runner script just starts the application and assume the moment later the app is ready to go, which is not the case for fabric-bridge and fabric-admin running together

Reply:
I think that we can change that as a follow up, but not today, if I may. I can add an option to search for some "needle" text in the app output in order to start test script.